js/app.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.menu');
+const navMenu = document.querySelector('.flex-nav-list');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+navMenu.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
```

Comment:
[OPTIONAL] Please use the `DRY` principle your code is not `DRY`. `DRY` stand for Don't Repeat Yourself, a basic principle of software development aimed at reducing the repetition of information. This requirement is optional because the previous reviewer miss this requirement so kindly don't skill this requirement because this recommended requirement :slightly_smiling_face: check below :point_down: my suggestion :+1: 

```suggestion
const menuButton = document.querySelector('.menu');
const navMenu = document.querySelector('.flex-nav-list');

const toggleMenu = () => {
  menuButton.classList.toggle('active');
  navMenu.classList.toggle('active');
}

menuButton.addEventListener('click', toggleMenu);
navMenu.addEventListener('click', toggleMenu);
```

- The first change is to rename the hamburger variable to menuButton to make the purpose of the variable more clear. Additionally, we can extract the anonymous function that toggles the "active" class on both elements into a named function toggleMenu() to make the code more readable.

- We can then pass this named function as the event listener to both the menuButton and navMenu elements.

- Finally, we remove the arrow function from the event listeners and simply pass the toggleMenu function as the argument. This makes the code more concise and easier to read.